includes/discovery/sensors/current/sentry3.inc.php

```diff
@@ -1,105 +0,0 @@
-<?php
-
-$divisor = '100';
-$outlet_divisor = $divisor;
-$multiplier = '1';
-
-// These PDUs may have > 1 "tower" accessible via a single management interface
-$tower_count = snmp_get($device, 'systemTowerCount.0', '-Ovq', 'Sentry3-MIB');
-$towers = 1;
-while ($towers <= $tower_count) {
-    // Check for Infeeds
-    $infeed_oids = snmp_walk($device, "infeedID.$towers", '-Osqn', 'Sentry3-MIB');
-    d_echo($infeed_oids . "\n");
-
-    $infeed_oids = trim($infeed_oids);
-
-    if ($infeed_oids) {
-        echo 'ServerTech Sentry Infeed ';
-    }
-
-    foreach (explode("\n", $infeed_oids) as $infeed_data) {
-        $infeed_data = trim($infeed_data);
-        if ($infeed_data) {
-            [$infeed_oid,$descr] = explode(' ', $infeed_data, 2);
-            $split_oid = explode('.', $infeed_oid);
-            $infeed_index = $split_oid[count($split_oid) - 1];
-
-            // infeedLoadValue
-            $infeed_oid = '.1.3.6.1.4.1.1718.3.2.2.1.7.' . $towers . '.' . $infeed_index;
-
-            $descr_string = snmp_get($device, "infeedID.$towers.$infeed_index", '-Ovq', 'Sentry3-MIB');
-            $descr = "Infeed $descr_string";
-            $low_warn_limit = null;
-            $low_limit = null;
-            $high_warn_limit = snmp_get($device, "infeedLoadHighThresh.$towers.$infeed_index", '-Ovq', 'Sentry3-MIB');
-            $high_limit = snmp_get($device, "infeedCapacity.$towers.$infeed_index", '-Ovq', 'Sentry3-MIB');
-            $current = (snmp_get($device, "$infeed_oid", '-Ovq', 'Sentry3-MIB') / $divisor);
-
-            if ($current >= 0) {
-                $tmp_index = "infeedID.$towers.$infeed_index";
```

Comment:
Hi @gunkaaa 
$tmp_index contain the index that will be used in LibreNMS database. If you keep it, then the historical data won't be lost. 
here, you would put in the YAML this : 
`index: 'infeedID.{{ $subindex1 }}.{{ $subindex0 }}'` 
which means
'concatenate 'infeedID.', the 2nd subindex value (towers), '.', and the 1st subindex value (infeed_index) if your YAML description was correct.

Reply:
Thanks, I've done this for the sensors that had existing handling in PHP.

It does mean that we don't pick up voltage for additional feeds (->phases) on each tower, but as they're manually configured it's no big loss.

Reply:
You can create the new ones anyway you like in term of ID. The only thing to do is to create the old one in YAML the same way as the old ones in PHP so they do not disappear and loose their history. That's the role of sensor_index to identify a sensor.